testplan/common/utils/sockets/fix/client.py

```diff
@@ -107,7 +107,7 @@ def _populate_tags(self, msg):
 
         msg[34] = self.out_seqno
         self.out_seqno += 1
-        if msg[35] == b'4':
+        if msg[35] in (b'4', u'4'):
```

Comment:
If msg values may be either bytes or unicode, that is an ugly interface. Can we fix that? If not, prefer use of `six.ensure_binary()` here.